common/src/main/java/org/redlance/dima_dencep/mods/rrls/mixins/LoadingOverlayMixin.java

```diff
@@ -75,6 +82,37 @@ private static int replaceAlpha(int color, int alpha) {
                     minecraft.font, ConfigExpectPlatform.reloadText(), i / 2, 70,
                     ConfigExpectPlatform.rgbProgress() ? ThreadLocalRandom.current().nextInt(0, 0xFFFFFF) : -1
             );
+
+            case Type.TEXT_WITH_BACKGROUND -> {
+                if (textWidget == null) {
+                    textWidget = new FocusableTextWidget(i, Component.literal(ConfigExpectPlatform.reloadText()), minecraft.font, 12);
+                }
+
+                textWidget.setX(i / 2 - textWidget.getWidth() / 2);
+                textWidget.setY(j - j / 3);
+
+                var shaderColor = RenderSystem.getShaderColor();
+                shaderColor = new float[] {
+                        shaderColor[0], shaderColor[1], shaderColor[2], shaderColor[3]
+                }; //Deep copy so these value won't change along with the call below.
+
+                // RGB
+                if (ConfigExpectPlatform.rgbProgress()) {
+                    var color = ThreadLocalRandom.current().nextInt(0, 0xFFFFFF);
+                    RenderSystem.setShaderColor((float) ((color >> 16) & 255) / 255, (float) ((color >> 8) & 255) / 255, (float) (color & 255) / 255, shaderColor[3]);
```

Comment:
`setShaderColor` is the only solution?

Reply:
`FocusableTextWidget` has method `setColor`

![image](https://github.com/user-attachments/assets/217fc3e8-a358-4d41-b6ed-3d6e3c7b9002)

Reply:
Not the only solution, but it can also change the border color, so I used it